prog/vm/nexus.rb

```diff
@@ -162,6 +162,14 @@ def before_run
         hop_destroy
       end
     end
+
+    when_recreate_set? do
+      if strand.label != "recreate"
```

Comment:
@furkansahin I know you like push, and probably not the time to do something about it, but it's in conflict with code like this.